src/queries/enum_structvariant_field_missing.ron

```diff
@@ -0,0 +1,66 @@
+SemverQuery(
+    id: "enum_structvariant_field_missing",
+    human_readable_name: "pub enum structvariant's field removed or renamed" ,
+    description: "An enum variant that is a struct has a field that is no longer available under its prior name.",
+    required_update: Major,
+    reference_link: None,
```

Comment:
Struct variant fields are still items the same way that struct fields are items, so the `item-removed` link that's in a few of the other lints is also appropriate here.